app/src/org/commcare/activities/UpdateActivity.java

```diff
@@ -416,6 +421,17 @@ private static CustomProgressDialog generateNormalUpdateInstallDialog(int taskId
         return dialog;
     }
 
+    private static void reportAppUpdate() {
+        try {
+            String username = CommCareApplication.instance().getRecordForCurrentUser().getUsername();
+            String updateLogMessage = "User " + username + " updated to app version " +
+                    ReportingUtils.getAppBuildNumber();
+            Logger.log(AndroidLogger.TYPE_USER, updateLogMessage);
+        } catch (SessionUnavailableException e) {
+            // Must be updating from the app manager, in which case we don't have a current user
```

Comment:
Wouldn't it still be worth reporting in this case, just without the username?

We'll still see it by device ID in the reports